docs/sources/shared/flow/reference/components/http-client-config-block.md

```diff
@@ -10,7 +10,7 @@ Name | Type | Description | Default | Required
 `bearer_token_file` | `string` | File containing a bearer token to authenticate with. | | no
 `proxy_url` | `string` | HTTP proxy to proxy requests through. | | no
 `follow_redirects` | `bool` | Whether redirects returned by the server should be followed. | `true` | no
-`enable_http_2` | `bool` | Whether HTTP2 is supported for requests. | `true` | no
+`enable_http2` | `bool` | Whether HTTP2 is supported for requests. | `true` | no
```

Comment:
this was documented incorrectly compared to the river type. Alternatively, I can update the river type to match the docs if we prefer `enable_http_2`

Reply:
No, I think `enable_http2` is probably what was intended.